ui_framework/src/components/dial_menu/_dial_menu.scss

```diff
@@ -0,0 +1,32 @@
+// Dial menu is a phone like dial comprised of an svg icon and a title.
+
+@include component('kuiDialMenu') {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: $kuiDialMenuWidth * 3; // Default to grid of 3
+
+  @include child('item') {
+    padding: $kuiSize;
+    height: $kuiDialMenuWidth;
+    width: $kuiDialMenuWidth;
+    display: block;
+    color: $kuiColorDarkShade;
+    border: $kuiBorderThin;
+    border-color: transparent;
+    border-radius: $kuiBorderRadius;
+
+    svg {
```

Comment:
Can we create a class here instead, `child('itemIcon')`? So we can avoid depending upon the element itself, and to avoid creating a nested selector.